kodilla-rps/src/main/java/GameProcessor.java

```diff
@@ -0,0 +1,70 @@
+public class GameProcessor {
+
+    private Player player;
+    private Computer computer;
+    private InputtingData inputtingData;
+    private int winsToEnd;
```

Comment:
maybe 'requiredWind' will be better name

Reply:
done